modules/core/src/main/java/org/apache/ignite/internal/processors/odbc/ClientListenerProcessor.java

```diff
@@ -646,5 +650,19 @@ private static boolean isNotDefault(ClientConnectorConfiguration cliConnCfg) {
 
             return false;
         }
+
+        /** {@inheritDoc} */
+        @Override public void showFullStack(boolean show) {
+            IgniteCompute compute = ctx.grid().compute(ctx.grid().cluster().forServers());
+
+            compute.broadcast(new CA() {
```

Comment:
Sending of non-static class over the network leads to serialization of a parent class.
Please, make it static.

Reply:
got it !